### DISCUSSION TITLE
Dist-upgrade readiness

### DISCUSSION THREAD

**Discussion by wasantosfi** [elementary/triage#491] [Ideas]
Maybe a good part of users would appreciate if eOS 8 comes with dist-upgrade readiness, in order to be able to upgrade system versions without the need to format the SSD all over again and again each time a new version is published.

**Comment by 439bananas** [comment]
This is probably the only thing holding me back from installing Elementary OS

**Comment by danirabbit** [comment]
Closing as a duplicate of https://github.com/orgs/elementary/discussions/346
